region/brinstar/kraid/Warehouse Energy Tank Room.json

```diff
@@ -96,17 +96,13 @@
             {"or": [
               "h_canUseMorphBombs",
               {"and": [
+                "Morph",
                 {"resourceAvailable": [{"type": "Energy", "count": 50}]},
                 {"or": [
                   {"ammo": {"type": "Missile", "count": 1}},
                   {"ammo": {"type": "Super", "count": 1}},
```

Comment:
I cant picture why the morph requirement was added. Maybe just needs the PB helper in the or that also includes morph?
They dont drop supers, so you could really only get 4 PBs, but maybe that's not as much of a concern?

Reply:
It's a disassembled `h_canUsePowerBombs`.  Spend an ammo of some kind to for sure get 1 PB and use that to generate full PBs.  Morph plus the PB means you can use Power Bombs.
- Without morph you can trade 1 missile or 1 super for 4 pbs but we can't write that as a repeatable strat to reach `{refill:PowerBomb}`
- The beetom who jumps on you when you enter the room isn't modeled that well, still.

Reply:
Oh, thanks, that makes sense now.